artemis-docker/prepare-docker.sh

```diff
@@ -124,7 +124,7 @@ if [ -n "${FROM_RELEASE}" ]; then
 
   cleanup
   
-  ARTEMIS_BASE_URL="$(curl -s https://www.apache.org/dyn/closer.cgi\?preferred=true)activemq/activemq-artemis/${ARTEMIS_VERSION}/"
+  ARTEMIS_BASE_URL="https://archive.apache.org/dist/activemq/activemq-artemis/${ARTEMIS_VERSION}/"
```

Comment:
```suggestion
  ARTEMIS_BASE_URL="$(curl -s https://www.apache.org/dyn/closer.cgi\?preferred=true)activemq/activemq-artemis/${ARTEMIS_VERSION}/"
  if ! curl --output /dev/null --silent --head --fail "${ARTEMIS_BASE_URL}"; then
    ARTEMIS_BASE_URL="https://archive.apache.org/dist/activemq/activemq-artemis/${ARTEMIS_VERSION}/"
  fi
```
The mirrored url should preferred if it exists.